packages/dialtone-vue2/components/tooltip/tooltip.vue

```diff
@@ -299,6 +314,7 @@ export default {
         offset: this.offset,
         interactive: false,
         trigger: 'manual',
+        appendTo: this.appendTo === 'body' ? this.anchorEl?.getRootNode()?.querySelector('body') : this.appendTo,
```

Comment:
Could we move this to a computed property?, it's kind of too much to have it in the template, this for vue3 also please